theme/common.less

```diff
@@ -1682,6 +1699,20 @@ p.ui.font.small {
     }
 }
 
+/* Thin view, for navbar size management */
+@media only screen and (max-width: @mediumScreenWidth) {
```

Comment:
hmm i would slightly prefer not to add another breakpoint (just because it's another view we'll have to remember to test), is it possible to reduce the padding on the button or the spacing between the sign in button and the settings gear at all?